pkg/pb/satellite.proto

```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+syntax = "proto3";
+option go_package = "pb";
+
+package satellite;
+
+service Satellite {
+    rpc Health(FileFealthRequest) returns (FileHealthResponse) {}
+}
+  
+message FileFealthRequest {
+    bytes file_id = 1;
```

Comment:
What do you mean by file_id? I think we are not using `file` in names around satellite.